src/include/btmem.h

```diff
@@ -420,6 +420,13 @@ struct __wt_page_modify {
     uint64_t rec_max_txn;
     wt_timestamp_t rec_max_timestamp;
 
+    /*
+     * Track the timestamp used for the most recent reconciliation. It's useful to avoid duplicating
+     * work when precise checkpoints are enabled, so we don't re-reconcile pages when no new content
+     * could be written.
+     */
+    wt_timestamp_t rec_pinned_stable_timestamp;
```

Comment:
Wouldn't a name like `last_reconcile_timestamp` be a bit more meaningful? The fact that it is set from the pinned stable timestamp that reconciliation started with does not feel like the variable's most salient property.

Reply:
I started with this change there, but it's actually the checkpoint timestamp that was in the system when the prior reconciliation happened that matters for disaggregated storage - since it is a movement in that time that makes it feasible for a reconciliation to include more information than the prior one.

Reply:
Oh, I see. That makes sense with the explanation, perhaps we should add a comment.